storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -14,7 +14,7 @@ but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
-You should have received a copy of the GNU General Public License
+You should have received a copy of the opt_tables_compatibility_checkGNU General Public License
```

Comment:
Here is broken.

Reply:
Ok, I will make it right. Just rollback it?

Reply:
O... IDE err